tmt/utils.py

```diff
@@ -554,6 +554,23 @@ def fail(self, message: str, shift: int = 0) -> None:
         """ Show a red failure message on info level, send to stderr """
         self.info('fail', message, color='red', shift=shift, err=True)
 
+    def _command_verbose_logger(
+            self,
+            key: str,
+            value: Optional[str] = None,
+            color: Optional[str] = None,
+            shift: int = 2,
+            level: int = 3,
+            err: bool = False) -> None:
+        """
+        Reports the executed command in verbose mode.
+
+        This is a tailored verbose() function used for command logging where
+        default parameters are adjusted (to preserve the function type).
+        """
+        self.verbose(key=key, value=value, color=color, shift=2,
+                     level=3, err=err)
```

Comment:
Since both `shift` and `level` remained as method parameters, just with their special defaults, I'd propose using the parameters here as well. Users can now call `_command_verbose_logger(shift=4)`, it'd be a shame if the value would be ignored.

Reply:
omg, of course.

Reply:
Fixed

Reply:
Seems that both `shift` and `level` values are still hardcoded here and do not respect the parameters?

Reply:
Fixed in 935cd819.